scripts/ci/automation_full_test.py

```diff
@@ -562,20 +562,20 @@ def run_instance_modules(self, instance_modules):
                 serial_tests.append(k)
             else:
                 parallel_tests.append(k)
-        pipeline_result = build_pipeline_result()
+        pipeline_result = build_pipeline_result() if job_name == 'FullTest' else None
         if serial_tests:
             azdev_test_result_fp = os.path.join(azdev_test_result_dir, f"test_results_{python_version}_{profile}_{instance_idx}.serial.xml")
             cmd = ['azdev', 'test', '--no-exitfirst', '--verbose', '--series'] + serial_tests + \
                   ['--profile', f'{profile}', '--xml-path', azdev_test_result_fp, '--pytest-args', '-o junit_family=xunit1 --durations=10 --tb=no']
             serial_error_flag = process_test(cmd, azdev_test_result_fp, live_rerun=fix_failure_tests)
-            pipeline_result = get_pipeline_result(azdev_test_result_fp, pipeline_result)
+            pipeline_result = get_pipeline_result(azdev_test_result_fp, pipeline_result) if job_name == 'FullTest' else None
         if parallel_tests:
             azdev_test_result_fp = os.path.join(azdev_test_result_dir, f"test_results_{python_version}_{profile}_{instance_idx}.parallel.xml")
             cmd = ['azdev', 'test', '--no-exitfirst', '--verbose'] + parallel_tests + \
                   ['--profile', f'{profile}', '--xml-path', azdev_test_result_fp, '--pytest-args', '-o junit_family=xunit1 --durations=10 --tb=no']
             parallel_error_flag = process_test(cmd, azdev_test_result_fp, live_rerun=fix_failure_tests)
-            pipeline_result = get_pipeline_result(azdev_test_result_fp, pipeline_result)
-        save_pipeline_result(pipeline_result)
+            pipeline_result = get_pipeline_result(azdev_test_result_fp, pipeline_result) if job_name == 'FullTest' else None
+        save_pipeline_result(pipeline_result) if job_name == 'FullTest' else None
```

Comment:
It's fine to collect and save pipeline result for jobs other than 'FullTest'. It depends on other job owner whether to publish these results or not